src/Evervault/EvervaultHttp.php

```diff
@@ -185,4 +259,16 @@ public function decrypt($data) {
         ], [], true);
         return $response->data;
     }
+
+    public function createToken($action, $data, $expiry) {
+        $payload = array(
+            'action' => $action,
+            'payload' => $data,
+            'expiry' => $expiry,
+        );
+
+        $response = $this->_makeCreateTokenRequest($payload);
```

Comment:
This is not needed. You can just rely on `_makeApiRequest`, similary to how it's done in `decrypt`